universal-login-wallet/src/ui/App.tsx

```diff
@@ -10,8 +10,8 @@ const App = () => (
     <div>
       <Navigation />
       <Switch>
-        <Route exact path="/" component={Home}/>
-        <Route path="/dogs" component={Dogs}/>
+        <Route exact={true} path='/' component={Home}/>
```

Comment:
Maybe configure linter so that it allows for the shorthand

Reply:
Done ✅